<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastre_about_form</class>
 <widget class="QDialog" name="cadastre_about_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastre - A propos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>610</width>
        <height>1195</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>Contributeurs</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Les structures suivantes ont participé au code :</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="1">
             <widget class="QLabel" name="label_logo_mtes">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_12">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>&lt;a href=&quot;https://www.ecologie.gouv.fr/&quot;  style=&quot;color:#434343;text-decoration:none;&quot;&gt;Ministère de la Transition Écologique&lt;/a&gt;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>&lt;a href=&quot;https://metropole.rennes.fr/&quot;  style=&quot;color:#434343;text-decoration:none;&quot;&gt;Rennes Métropole&lt;/a&gt;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_logo_rennes_metropole">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Financeurs</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Les évolutions 2020 de l'extension Cadastre ont été financés par :</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="3" column="0">
             <widget class="QLabel" name="label_logo_datagences_bretagne">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_23">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>&lt;a href=&quot;http://asadefrance.fr/&quot;  style=&quot;color:#434343;text-decoration:none;&quot;&gt;ASA de France&lt;/a&gt;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_logo_cd54">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="label_26">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>&lt;a href=&quot;https://www.datagences-bretagne.bzh/&quot;  style=&quot;color:#434343;text-decoration:none;&quot;&gt;Agences d'Urbanisme de Bretagne&lt;/a&gt; et autres : &lt;a href=&quot;https://www.agam.org/&quot;  style=&quot;color:#434343;text-decoration:none;&quot;&gt;AGAM&lt;/a&gt;, &lt;a href=&quot;http://www.ausb.org&quot;  style=&quot;color:#434343;text-decoration:none;&quot;&gt;AUSB&lt;/a&gt;, &lt;a href=&quot;http://www.aduga.org/&quot;  style=&quot;color:#434343;text-decoration:none;&quot;&gt;ADUGA&lt;/a&gt;, &lt;a href=&quot;https://audat.org/&quot;  style=&quot;color:#434343;text-decoration:none;&quot;&gt;AUDAT&lt;/a&gt;, &lt;a href=&quot;https://clermontmetropole.org/&quot;  style=&quot;color:#434343;text-decoration:none;&quot;&gt;AUD CM&lt;/a&gt;, &lt;a href=&quot;http://www.agape-lorrainenord.eu/&quot;  style=&quot;color:#434343;text-decoration:none;&quot;&gt;AGAPE LN&lt;/a&gt;, &lt;a href=&quot;http://www.boulogne-developpement.com/&quot;  style=&quot;color:#434343;text-decoration:none;&quot;&gt;BMD CO&lt;/a&gt;</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="label_22">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>&lt;a href=&quot;http://www.meurthe-et-moselle.fr/&quot;  style=&quot;color:#434343;text-decoration:none;&quot;&gt;Conseil Départemental de Meuthe et Moselle&lt;/a&gt;</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_logo_asadefrance">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_logo_mtes_2">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_9">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>&lt;a href=&quot;https://www.ecologie.gouv.fr/&quot;  style=&quot;color:#434343;text-decoration:none;&quot;&gt;Ministère de la Transition Écologique&lt;/a&gt;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_logo_grandnarbonne">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label_25">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>&lt;a href=&quot;https://www.legrandnarbonne.com/&quot;  style=&quot;color:#434343;text-decoration:none;&quot;&gt;Communauté d'Agglomération du Grand Narbonne&lt;/a&gt;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>La réalisation de l'extension Cadastre a été financée initialement par :</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_logo_ue">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_logo_feder">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_logo_aduga">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>&lt;a href=&quot;http://www.aduga.org/&quot; style=&quot;color:#434343;text-decoration:none;&quot;&gt;L'Agence de Développement et  d'Urbanisme du Grand Amiénois&lt;/a&gt;</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>&lt;a href=&quot;http://www.picardie-europe.eu/&quot; style=&quot;color:#434343;text-decoration:none;&quot;&gt;Le  Fonds Européen de Développement Régional de Picardie&lt;/a&gt;</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>&lt;a href=&quot;http://europa.eu/index_fr.htm&quot; style=&quot;color:#434343;text-decoration:none;&quot;&gt;L'Union Européenne&lt;/a&gt;</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_logo_picardie">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>&lt;a href=&quot;http://www.picardie.fr/&quot; style=&quot;color:#434343;text-decoration:none;&quot;&gt;Le Conseil Régional de Picardie&lt;/a&gt;</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Maintenance</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Les structures suivantes ont aidé 3Liz pour la maintenance ou l'ajout de fonctionnalités:&lt;/p&gt;&lt;p&gt;- L'&lt;span style=&quot; font-weight:600;&quot;&gt;ADUGA&lt;/span&gt; : maintenace pour les millésimes MAJIC&lt;/p&gt;&lt;p&gt;- La &lt;span style=&quot; font-weight:600;&quot;&gt;Ville de Megève&lt;/span&gt;: migration de l'extension vers QGIS 3&lt;/p&gt;&lt;p&gt;- La &lt;span style=&quot; font-weight:600;&quot;&gt;Communauté d'Agglomération du Soissonnais&lt;/span&gt; : amélioration de la recherche et de la fiche parcellaire&lt;/p&gt;&lt;p&gt;- Le &lt;span style=&quot; font-weight:600;&quot;&gt;Conseil départemental du Gard&lt;/span&gt; : ajout d'informations dans la fiche parcellaire&lt;/p&gt;&lt;p&gt;- La &lt;span style=&quot; font-weight:600;&quot;&gt;Métropole de Rennes&lt;/span&gt; : ajout des supports 2019 et 2020 des millésimes MAJIC&lt;/p&gt;&lt;p&gt;- le &lt;span style=&quot; font-weight:600;&quot;&gt;Ministère de l'Écologie&lt;/span&gt; : détails sur les propriétaires (indivisions)&lt;/p&gt; - La &lt;span style=&quot; font-weight:600;&quot;&gt;Métropole de Brest Océane&lt;/span&gt; : enrichissement de la recherche de propriétaires (par nom d'usage, par nom de naissance, par commune)&lt;/p&gt;&lt;p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="textFormat">
             <enum>Qt::RichText</enum>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Conception</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>L'extension Cadastre a été conçu et développé par la Société 3Liz</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_logo_3liz">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>&lt;a href=&quot;https://www.3liz.com&quot; style=&quot;color:#7BA11A;text-decoration:none;&quot;&gt;Libérez vos SIG !&lt;/a&gt;</string>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Auteurs</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Michaël DOUCHIN</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;mailto:info@3liz.com?subject=Extension cadastre - A propos&quot;&gt;&lt;span style=&quot; color:#7ba11a;&quot;&gt;info@3liz.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Sources</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Dépôt sur GitHub</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>&lt;a href=&quot;https://github.com/3liz/QgisCadastrePlugin&quot; style=&quot;color:#7BA11A;text-decoration:none;&quot;&gt;https://github.com/3liz/QgisCadastrePlugin&lt;/a&gt;</string>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Licence</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Licence GPL Version 2.0</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>&lt;a href=&quot;https://www.gnu.org/licenses/old-licenses/gpl-2.0.fr.html&quot; style=&quot;color:#7BA11A;text-decoration:none;&quot;&gt;Voir le site officiel&lt;/a&gt;</string>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Ressources</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Les scripts d'import pour PostGreSQL proviennent initialement de l'outil OpenCadastre (licence GPL). Ils ont été adaptés et améliorés pour leur utilisation dans cette extension. Nos remerciement aux contributeurs.</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>&lt;a href=&quot;https://adullact.net/scm/viewvc.php/trunk/data/pgsql/?root=opencadastre&quot;  style=&quot;color:#7BA11A;text-decoration:none;&quot;&gt;Dépôt OpenCadastre&lt;/a&gt;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
